src/Network/Network/help/New-AzFirewallThreatIntelWhitelist.md

```diff
@@ -0,0 +1,129 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/new-azfirewallthreatintelwhitelist
+schema: 2.0.0
+---
+
+# New-AzFirewallThreatIntelWhitelist
+
+## SYNOPSIS
+Create a new threat intelligence whitelist for Azure Firewall
+
+## SYNTAX
+
+```
+New-AzFirewallThreatIntelWhitelist [-FQDN <String[]>] [-IpAddress <String[]>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzFirewallThreatIntelWhitelist** cmdlet creates a threat intel whitelist for Azure Firewall.
```

Comment:
Let's make it clear what this cmelet does
```suggestion
The **New-AzFirewallThreatIntelWhitelist** cmdlet creates a threat intel whitelist object, which can be used when creating or setting an Azure Firewall.
```

Reply:
changed